code/modules/mob/living/living_eating.dm

```diff
@@ -2,8 +2,15 @@
 	user = user || src
 	if(get_food_satiation(consumption_method) < get_max_nutrition())
```

Comment:
uh as an aside, isn't this wrong? it'd compare hydration against max nutrition instead of max hydration. maybe calls for adding `get_food_max_satiety(consumption_method)`?

Reply:
get_food_satiation returns current hydration/nutrition level, get_max_nutrition is only nutrition, is the point